airflow/README.md

```diff
@@ -285,6 +290,26 @@ For containerized environments, see the [Autodiscovery Integration Templates][8]
 
 Ensure that `url` matches your Airflow [webserver `base_url`][19], the URL used to connect to your Airflow instance. Replace `localhost` with the template variable `%%host%%`.
 
+If you are using Airflow's Helm chart, this [exposes the webserver as a ClusterIP service][22] that you should use in the `url` parameter. 
+
+For example, your Autodiscovery annotations may look like the following:
+
+```
+apiVersion: v1
+kind: Pod
+# (...)
+metadata:
+  name: '<POD_NAME>'
+  annotations:
+    ad.datadoghq.com/<CONTAINER_IDENTIFIER>.checks: |
+      {
+        "airflow": {
+          "instances": ["url": "http://airflow-ui.%%kube_namespace%%.svc.cluster.local:8080"]
```

Comment:
Interesting. Does `%%kube_namespace%%` work out of the box like that? Or is this more of a place holder?

Reply:
yup, that's an Autodiscovery template variable (see: https://docs.datadoghq.com/containers/guide/template_variables/)

Reply:
👍 TIL.